src/env/default.rs

```diff
@@ -97,6 +102,21 @@ impl LogFd {
         Ok(readed)
     }
 
+    pub fn read_aio(&self, seq: usize, ctx: &mut AioContext, offset: u64) {
+        let mut buf = ctx.buf_vec.last().unwrap().lock().unwrap();
```

Comment:
The lock is meaningless. The mutable reference (`&mut AioContext`) ensures there's only one thread accessing the ctx.

Of course, due to the unsafe block, the `buf_vec` is in fact "leaked" to aio code. That's why we must manually guarantee there's no one reading or writing to `buf_vec` after calling `aio_read`.